spec/controllers/api/v1/recipes_controller_spec.rb

```diff
@@ -54,7 +56,10 @@
   describe '[PUT] #update' do
     let(:old_name) { 'Rosol' }
     let(:new_name) { 'Pierogi' }
-    let(:recipe) { create(:recipe, name: old_name) }
+    let(:old_video_link) { 'https://www.youtube.com/watch?v=60KwNYk09BU' }
+    let(:new_video_link) { 'https://www.youtube.com/watch?v=74re1sJJFtg' }
+    let(:recipe) { create(:recipe, name: old_name, video_link: old_video_link) }
+    let(:recipeJSON) { JSON.parse(response.body) }
```

Comment:
Ruby uses snake case not camel case to name variables
```suggestion
    let(:recipe_json) { JSON.parse(response.body) }
```